modules/rewrite/classes/RewriteUrlGenerator.class

```diff
@@ -43,6 +43,9 @@ class RewriteUrlGenerator extends GalleryUrlGenerator {
      */
     var $_shortUrls = array();
 
+    /* deprecated dynamic properties in php 8.2 */
+    var $_language;
```

Comment:
I don't think this is needed?

Reply:
Without the variable declaration unit test 1773 fails with the deprecation warning from php 8.2